### PR TITLE
#75 Fix menu disappearing after resizing the window

### PR DESCRIPTION
**Pull request by Everkers**
## Solution
Add menu-close class to the left menu that will have effect only on 900px breakpoint 

### BEFORE
<a href="https://www.loom.com/share/f5b6ccbebcb14e1f90e924d14a3cf8cb"> <p>Baat Cheet Sidebar bug</p>  </a>
### AFTER
<a href="https://www.loom.com/share/7510670f728042b39c8d427f90ca1f8e"> <p>Baat Cheet Sidebar bug fix</p>  </a>

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -383,9 +383,11 @@ body {
   .sidebar-btn {
     visibility: visible;
   }
-
+  .close-menu{
```
Comment (LalitNM) [inline public/style.css]: Change this to `menu-closed`.

File: public/style.css [hunk 2]
```diff
@@ -383,9 +383,11 @@ body {
...
+    left:-300px !important;
+  }
   .left {
-    left: -300px;
+    left: 0px;
   }
```
Comment (LalitNM) [inline public/style.css]: `left` is already a class, you can change this to `open-menu`

**Review by LalitNM — CHANGES_REQUESTED** [review]
In addition to requested changes. Remove `package-lock.json` from your commit, as this is just installation file.

**Comment by Everkers** [comment]
@LalitNM done ✅

File: public/style.css [hunk 3]
```diff
@@ -383,11 +383,15 @@ body {
...
-    left: -300px;
+  .menu-closed{
+    left:-300px !important;
+  }
+  .open-menu {
+    left: 0px !important;
+  }
+  .left{
+    left:-300px;
   }
```
Comment (LalitNM) [inline public/style.css:394]: This is still here, remove it.

**Review by LalitNM — CHANGES_REQUESTED** [review]

**Comment by LalitNM** [comment]
And once you have done requested changes, please clean git history, we want only one commit in this PR.

File: public/style.css (hunk 3 above)
Comment (Everkers) [reply public/style.css:394]: it's still there to keep the menu closed by default in mobile screens

**Comment by LalitNM** [comment]
@Everkers as you can see that this PR has 6 commits, some of which are already there in `KamandPrompt: master`
Now you have two ways to solve this problem.

1. Solve problem in this PR

- reset this branch(`Everkers:master`) to a state before creating this PR.

- add remote upstream to your local gir repository using `git remote add upstream <link to upstream>`

- rebase this branch to `upstream: master` using `git rebase upstream/master`

- add your changes in one commit only

- force push this branch here using `git push -f origin: master`

2. Close this PR and and delete your fork, again fork, and make PR after adding changes.

**Comment by Everkers** [comment]
> @Everkers as you can see that this PR has 6 commits, some of which are already there in `KamandPrompt: master`
> Now you have two ways to solve this problem.
> 
> 1. Solve problem in this PR
> 
> * reset this branch(`Everkers:master`) to a state before creating this PR.
> * add remote upstream to your local gir repository using `git remote add upstream <link to upstream>`
> * rebase this branch to `upstream: master` using `git rebase upstream/master`
> * add your changes in one commit only
> * force push this branch here using `git push -f origin: master`
> 
> 1. Close this PR and and delete your fork, again fork, and make PR after adding changes.

@LalitNM, I have created a new pr, sorry for the delay 😄 !

**Pull request closed without merging**